packages/frontend/src/lib/ModelSelect.svelte

```diff
@@ -0,0 +1,84 @@
+<script lang="ts">
+import { faCheckCircle, faDownload } from '@fortawesome/free-solid-svg-icons';
+import Select from 'svelte-select';
+import Fa from 'svelte-fa';
+import type { ModelInfo } from '@shared/src/models/IModelInfo';
+
+export let disabled: boolean = false;
+/**
+ * Recommended model ids
+ */
+export let recommended: string[] | undefined = undefined;
+
+/**
+ * List of models
+ */
+export let models: ModelInfo[];
+
+let sorted: ModelInfo[];
+
+/**
+ * Put the recommended models at the top
+ */
+$: {
+  sorted = models.toSorted((a, b) => {
+    return (recommended?.includes(a.id) ? 0 : 1) - (recommended?.includes(b.id) ? 0 : 1);
+  });
```

Comment:
should downloaded models be sorted first as well ? I guess that if I have downloaded a model I don't want to be listed at the end.

Reply:
Good idea